.eslintignore

```diff
@@ -0,0 +1,5 @@
+node_modules/
```

Comment:
There is no need to add `node_modules` to `.eslintignore` since it is ignored by default.

Reply:
:thinking: hm, how did eslint manage to check the code in there locally...

Just tested locally, indeed fine. Strange, I probably misinterpreted the output for the checking of node_modules when it was for something else.

Reply:
I don't think ESLint ignores `node_modules` by default (at least, it didn't use to) if you have a `.eslintignore` file. But ESLint does use `.gitignore` instead of `.eslintignore` by default if the former is present and the latter is not.

Reply:
@aqrln according to ESLint documentation:
>In addition to any patterns in a `.eslintignore` file, ESLint always ignores files in `/node_modules/*` and `/bower_components/*`.

Which means that ESLint ignores `node_modules` by default regardless of the `.eslintignore` file existence.